src/porepy/models/contact_mechanics_model.py

```diff
@@ -405,7 +416,7 @@ def prepare_simulation(self) -> None:
             self.assembler: pp.Assembler = pp.Assembler(self.mdg, self.dof_manager)
 
         self._discretize()
-        self._initialize_linear_solver()
+        self._initialize_linear_solver()  # TODO: Purge method from standard models?
```

Comment:
```suggestion
        self._initialize_linear_solver()
```
My expectation is that we will finally make progress on this part in the coming months, so I suggest we keep the method, and hopefully see it grow.

Reply:
In that case, should we move to AbstractModel?